samples/web/content/apprtc/bigquery/analytics_schema.json

```diff
@@ -0,0 +1,27 @@
+[
+  {
+    "name": "impression_name",
```

Comment:
Make the python code read from this and reuse the strings? So we don't need to maintain the strings in multiple places.